src/components/ContentWidgets/ImagesInRow/ImagesInRow.tsx

```diff
@@ -16,24 +16,23 @@ type linkCaptionPair = {
  * July 14, 2019
  */
 export const ImagesInRow: React.FC<ContentSingularData> = ({
-    imagesInRow_linkCaptionPairs
+    imagesInRow_items
 }) => {
-    if (!imagesInRow_linkCaptionPairs) {
+    if (!imagesInRow_items) {
         return <></>;
     }
     return <>
         <div className="img-row">
-            {imagesInRow_linkCaptionPairs.map((pair: linkCaptionPair, index: number) => (
+            {imagesInRow_items.map((pair: imgRowItem, index: number) => (
                 <div className="img-row-item">
                     <figure>
                         <a href={pair.extLink ? pair.extLink : pair.imgLink}>
                             <img src={pair.imgLink} alt={"image" + index} />
```

Comment:
This alt tag is not aria compliant. I would suggest you instead have a box for the user to enter an alt tag and have the default be something. You never want to have "image" or "photo" be part of the alt tag. It is also not descriptive.

Reply:
You might even want it to be the image caption.